src/components/Display.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable class-methods-use-this */
+import React from 'react';
+
+export default class Display extends React.Component {
+  render() {
+    return (
+      <div style={{ border: 'dotted thick blue' }}>
+        {this.props.value}
+      </div>
+    );
+  }
```

Comment:
lets use propTypes to make sure the value coming in as props is a string and let's use defaultProps to set a default value of '0'

Reply:
I tried using them, but the linter complained about them being deprecated, that's why I omitted them. :-)

Reply:
Oh! Now I get it: I was using React.PropTypes as explained in one tutorial; that's why it was appearing as deprecated!

Reply:
Nice catch 👍